test/RewardsPoolInfinite.ts

```diff
@@ -287,27 +231,6 @@ describe('RewardsPoolBaseInfinite', () => {
       expect(await stakingToken.balanceOf(rewardsPoolBaseInfinite.address)).to.be.eq(amount);
     });
 
-    it('Should not be able to call onlyOwner functions', async () => {
```

Comment:
Why is this removed?

Reply:
Same thing as mention above, so i'll close this one